src/realm/sync/CMakeLists.txt

```diff
@@ -52,16 +52,19 @@ set(SYNC_INSTALL_HEADERS
     protocol.hpp
     subscriptions.hpp
     transform.hpp
-
-    network/http.hpp
 )
 
-set(NOINST_HEADERS
-    trigger.hpp
+set(SYNC_NETWORK_INSTALL_HEADERS
     network/default_socket.hpp
```

Comment:
Unless you really need this file (i.e. can't compile without it), I would recommend moving this one to the NOINST_HEADERS section, since it is intended to only be used internally.

Reply:
because `sync_client.hpp` uses that header we can't do that unfortunately.